scripts/menu.sh

```diff
@@ -107,6 +125,26 @@ function configure_aws() {
   export AWS_SECRET_ACCESS_KEY=$(inputbox "Amazon Web Services" "AWS Secret Key" "${AWS_SECRET_ACCESS_KEY}")
   export AWS_S3_BUCKET=$(inputbox "Amazon Web Services" "AWS S3 Bucket Name" "${AWS_S3_BUCKET}")
   export NAMESPACE=$(inputbox "Deepcell" "Cluster Name" "${NAMESPACE}")
+  export MASTER_MACHINE_TYPE=$(inputbox "Amazon Web Services" "Master Node Machine Type" "${MASTER_MACHINE_TYPE:-m4.large}")
+  export NODE_MACHINE_TYPE=$(inputbox "Amazon Web Services" "Worker Nodes Machine Type" "${NODE_MACHINE_TYPE:-m4.large}")
+
+  # let's just hardcode the menu, since all instance types are apparently available in all regions
+  # and there's no built-in way to list type in the aws-cli
+  local gpu_types="g3s.xlarge 1GPU OFF
```

Comment:
<img width="659" alt="screen shot 2018-11-15 at 3 28 46 pm" src="https://user-images.githubusercontent.com/7930703/48588268-3e00d280-e8eb-11e8-8677-3fb4efbc4d3e.png">

Reply:
Looks like I should have done more testing...

Reply:
Looks like the typo was indeed on the AWS website.  Changing `g3s.xlarge` to `g3.xlarge` allowed the cluster to be created.